lib/ttfunk/table/vorg.rb

```diff
@@ -0,0 +1,54 @@
+require_relative '../table'
+
+module TTFunk
+  class Table
+    class Vorg < Table
+      TAG = 'VORG'.freeze
+      ENTRY_SIZE = 4
```

Comment:
Is this used for anything?

Reply:
Good catch, these were left over from an earlier version of the table.